packages/vite/src/client/overlay.ts

```diff
@@ -123,6 +123,35 @@ export class ErrorOverlay extends HTMLElement {
     this.root = this.attachShadow({ mode: 'open' })
     this.root.innerHTML = template
 
+    if (!err) {
```

Comment:
Let's leave some comments here explaining this code path is used when there's a SSR error directly injecting the overlay in the returned HTML

Reply:
Added, thanks!